packages/login/src/vaadin-login-form.js

```diff
@@ -103,6 +105,8 @@ class LoginForm extends LoginFormMixin(ElementMixin(ThemableMixin(PolymerElement
         >
           [[i18n.form.forgotPassword]]
         </vaadin-button>
+
+        <div id="vaadinLoginFormFooter" slot="footer" style="display: contents;"></div>
```

Comment:
Actually, this element is unneeded as we might assign DOM elements to the `footer` slot directly.